plugins/plugin.proto

```diff
@@ -84,18 +96,29 @@ message PluginResponse {
   // status code.
   repeated string error = 1;
   
-  // text output
+  // text output, typically written to stdout by openapic.
```

Comment:
I made the above change in [this commit](https://github.com/googleapis/openapi-compiler/pull/23/commits/13c13622bff6744524bda9497e382c037da00e43). Now plugins only send errors and files back to the compiler, and they can also write to stderr.